engines/sci/engine/message.cpp

```diff
@@ -444,6 +444,14 @@ bool MessageState::stringStage(Common::String &outstr, const Common::String &inS
 		// SCI32 seems to support having digits in stage directions
 		if (((inStr[i] >= 'a') && (inStr[i] <= 'z')) || ((inStr[i] >= '0') && (inStr[i] <= '9') && (getSciVersion() < SCI_VERSION_2)))
 			return false;
+
+		// If it contains Hebrew letters, it's not a stage direction
+		// besides that point, keep the English logic
+		if (g_sci->getLanguage() == Common::HE_ISR)
```

Comment:
Please, enclose the code in this if statement in brackets - { }, in order to improve readability

Reply:
Done.